frontend/packages/console-plugin-sdk/src/codegen/plugin-integration-tests.ts

```diff
@@ -0,0 +1,39 @@
+import * as _ from 'lodash';
+import {
+  PluginPackage,
+  filterActivePluginPackages,
+  PluginPackageFilter,
+  resolvePluginPackages,
+} from './plugin-resolver';
+
+const CONSOLE_PLUGIN_PREFIX = '@console/';
+const CONSOLE_INEGRATION_TESTS_PREFIX = '@console/integration-tests/';
+
+/**
+ * Return a map of integration tests.
+ */
+export const pluginIntegrationTests = (
+  monorepoRootDir: string = process.cwd(),
+  pluginFilter: PluginPackageFilter = filterActivePluginPackages,
+) => {
+  const shortName = (plugin: PluginPackage) => plugin.name.slice(CONSOLE_PLUGIN_PREFIX.length);
+
+  const integrationTests = (plugin: PluginPackage) =>
+    plugin.name.startsWith(CONSOLE_PLUGIN_PREFIX) &&
```

Comment:
Console plugins with package names that start with `@console` is a convention, not a hard requirement.

Others can still clone the repo & add/link another package that fulfills the Console plugin contract. Plugin packages are scanned starting from `monorepoRootDir` which means they don't even have to be under `packages`, but this is again our structural convention.

Therefore, I'd suggest to remove this line.

Reply:
Thanks !
I needed it in some earlier iteration of the code, and forgot to remove it :+1: